examples/platform/telink/common/src/AppTaskCommon.cpp

```diff
@@ -103,6 +104,14 @@ Identify sIdentify = {
 uint8_t sTestEventTriggerEnableKey[TestEventTriggerDelegate::kEnableKeyLength] = { 0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77,
                                                                                    0x88, 0x99, 0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff };
 #endif
+
+class AppCallbacks : public AppDelegate
+{
+public:
+    void OnCommissioningWindowClosed() override { chip::DeviceLayer::Internal::BLEMgr().Shutdown(); }
```

Comment:
@bzbarsky-apple 

Thank you for your note, I have added a fuse, to prevent the session from being destroyed. 
Generally if the session is active, the commissioning window will not be closed.
This delegate was added to shut down the BLE loop if the BLE is not being used any more to save the battery life.